adops/setting-up-prebid-for-amp-in-dfp.md

```diff
@@ -34,64 +37,53 @@ In addition to your other line item settings, you'll need the following:
 
 + Set **Rotate creatives** to *Evenly*.
 
-+ In the targeting section, select **Key-values** targeting.  In [Show Prebid Ads on AMP Pages]({{site.github.url}}/dev-docs/show-prebid-ads-on-amp-pages.html), we targeted the "prebid_amp" keyword set to "true" from the developer side as an example, but you will want to coordinate with your development team to use your own key-values.
++ In the targeting section, select **Key-values** targeting.  You'll need to coordinate with your development team on what key-values you want to target.
 
 Save your line item and add a creative.
 
-
 ## Creative Setup
 
 On the new creative screen, select the **Third party** creative type.
 
-Enter the below code snippet in the **Code snippet** text area, and make sure to uncheck the **Serve into a SafeFrame** checkbox.
+Enter the below code snippet in the **Code snippet** text area, and *make sure to uncheck the **Serve into a SafeFrame** checkbox*.
```

Comment:
Please remove `*make sure to uncheck the **Serve into a SafeFrame** checkbox*.` SafeFrame is required to serve into AMP

Reply:
Removed in d682f1b , thanks